nqeit-cross-engine/src/test/java/org/projectnessie/integtests/crossengine/ITCrossEngineDremio.java

```diff
@@ -90,6 +199,28 @@ public static void setupEngines(
 
     Stream.of(SPARK_TABLE, DREMIO_TABLE, FLINK_TABLE)
         .forEach(x -> sparkSql("DROP TABLE IF EXISTS %s", sparkTable(x)));
+
+    selectEngines = asList(DREMIO_ENGINE, SPARK_ENGINE, FLINK_ENGINE);
+    insertEngines = asList(DREMIO_ENGINE, SPARK_ENGINE, FLINK_ENGINE);
+    deleteEngines = asList(DREMIO_ENGINE, SPARK_ENGINE); // flink does not support deletes
+
+    long seed = System.currentTimeMillis();
```

Comment:
nit: Would you mind making it settable to allow reproducing runs without recompilation?

Reply:
its quite cumbersome to pass it through gradle etc.

it seems unlikely that order will ever be the culprit but if we need to debug you'd probably do it with a branch and the automated job, instead of running manually locally...
so YAGNI imo